python-client/giskard/models/model_explanation.py

```diff
@@ -66,13 +140,52 @@ def prediction_function(_df):
 
 
 def _get_highest_proba_shap(shap_values: list, model: BaseModel, dataset: Dataset) -> list:
-    """Get SHAP explanations of classes with the highest predicted probability."""
+    """Get SHAP explanations of classes with the highest predicted probability.
+
+    Parameters
+    ----------
+    shap_values : list (n_classes x n_samples)
+        Raw SHAP contributions of the features from the `dataset` to the `model`
```

Comment:
remove "from the `dataset` to the `model`
        predictions."

Reply:
\+